leakcanary-analyzer/src/main/java/com/squareup/leakcanary/LeakReference.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.leakcanary;
+
+import java.io.Serializable;
+
+/**
+ * A single field in a {@link LeakTraceElement}.
+ */
+public final class LeakReference implements Serializable {
```

Comment:
Serializable 😭

Reply:
😢 The only other no dependency option would be parcelable, and I'm wayyy too lazy.

Reply:
(all that stuff has always been Serializable in leakcanary)

Reply:
and actually analyzer is a java project, so no Parcelable either.